src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.BinaryOperations.Combinations.ttinclude

```diff
@@ -0,0 +1,258 @@
+        new List<string> {"sbyte", "decimal", "decimal"},
+        new List<string> {"sbyte", "double", "double"},
+        new List<string> {"sbyte", "float", "float"},
+        new List<string> {"sbyte", "int", "int"},
+        new List<string> {"sbyte", "long", "long"},
+        new List<string> {"sbyte", "sbyte", "sbyte"},
+        new List<string> {"sbyte", "short", "short"},
+        new List<string> {"sbyte", "uint", "uint"},
+        new List<string> {"sbyte", "ulong", "ulong"},
+        new List<string> {"sbyte", "ushort", "ushort"},
+        new List<string> {"short", "byte", "short"},
+        new List<string> {"short", "decimal", "decimal"},
+        new List<string> {"short", "double", "double"},
+        new List<string> {"short", "float", "float"},
+        new List<string> {"short", "int", "int"},
+        new List<string> {"short", "long", "long"},
+        new List<string> {"short", "sbyte", "short"},
+        new List<string> {"short", "short", "short"},
+        new List<string> {"short", "uint", "uint"},
+        new List<string> {"short", "ulong", "ulong"},
+        new List<string> {"uint", "byte", "uint"},
+        new List<string> {"uint", "decimal", "decimal"},
+        new List<string> {"uint", "double", "double"},
+        new List<string> {"uint", "float", "float"},
+        new List<string> {"uint", "long", "long"},
+        new List<string> {"uint", "sbyte", "uint"},
+        new List<string> {"uint", "short", "uint"},
+        new List<string> {"uint", "uint", "uint"},
+        new List<string> {"uint", "ulong", "ulong"},
+        new List<string> {"uint", "ushort", "uint"},
+        new List<string> {"ulong", "byte", "ulong"},
+        new List<string> {"ulong", "decimal", "decimal"},
+        new List<string> {"ulong", "double", "double"},
+        new List<string> {"ulong", "float", "float"},
+        new List<string> {"ulong", "int", "ulong"},
+        new List<string> {"ulong", "sbyte", "ulong"},
+        new List<string> {"ulong", "short", "ulong"},
+        new List<string> {"ulong", "uint", "ulong"},
+        new List<string> {"ulong", "ulong", "ulong"},
+        new List<string> {"ulong", "ushort", "ulong"},
+        new List<string> {"ushort", "byte", "ushort"},
+        new List<string> {"ushort", "decimal", "decimal"},
+        new List<string> {"ushort", "double", "double"},
+        new List<string> {"ushort", "float", "float"},
+        new List<string> {"ushort", "int", "int"},
+        new List<string> {"ushort", "long", "long"},
+        new List<string> {"ushort", "sbyte", "ushort"},
+        new List<string> {"ushort", "uint", "uint"},
+        new List<string> {"ushort", "ulong", "ulong"},
+        new List<string> {"ushort", "ushort", "ushort"},
+    };
+}
+
+public class ComparisonOperationCombinations
+{
+    public static List<List<string>> comparisonOperationCombinations = new List<List<string>>
+    {
+        new List<string> {"bool", "bool", "bool"},
```

Comment:
Do you really need the `bool` at the end? Won't all comparison operations result in a `bool`?

Reply:
No we don't, but I found it helpful when reasoning about the text templates. Also, I'm just re-using the `TypeCombination` class from the binary ops now. I could change the code here to remove the 3rd "bool", but that would be a bunch of accompanying changes and result in the same code?